packages/ui/components/button/Button.tsx

```diff
@@ -210,7 +210,7 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
             <StartIcon
               className={classNames(
                 variant === "icon" && "h-4 w-4",
-                variant === "button" && "h-4 w-4 stroke-[1.5px] ltr:mr-2 rtl:ml-2"
+                variant === "button" && "h-4 w-4 stroke-[1.5px] ltr:ml-2 rtl:mr-2"
```

Comment:
I think this was the other way around as a bug, could see it in the "Purchase a License" button as anchor that includes am "external link" icon at the right of the button text.